docs/physics/tardisgamma/decayenergy.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    print("The total energy production rate for 1 solar mass of 56Ni after 10 days is:", energy_production_rate, "erg/s")
</pre><p>Probably use f-strings here rather than printing with multiple arguments. </p><p> </p><p>If you're not familiar, you can do print(f"text goes here {variable goes here} more text here). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I made this change!